src/plugins/vis_builder/public/application/components/data_tab/use/use_dropbox.tsx

```diff
@@ -60,6 +70,15 @@ export const useDropbox = (props: UseDropboxProps): DropboxProps => {
     [dropboxAggs, timeRange]
   );
 
+  useEffect(() => {
+    if (displayFields && JSON.stringify(fields) !== JSON.stringify(displayFields)) {
+      const newDisplayState = { ...schemaDisplayStates };
+      newDisplayState[schema.name] = displayFields;
+      setSchemaDisplayStates(newDisplayState);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why do you need to disable this? Also same comment about disabling eslint here. add a comment line justifying it.

Reply:
Added Comment